packages/p2p/src/components/my-ads/my-ads.scss

```diff
@@ -630,10 +638,6 @@
         }
     }
 
-    &__modal-body--scroll {
-        overflow: auto;
-    }
-
```

Comment:
`&__modal-body` is already declared in line 197, refactored `&__modal-body--scroll` to `&--scroll` within the `&__modal-body` block